src/views/all/Home.vue

```diff
@@ -179,15 +137,19 @@ export default {
   },
   apollo: {
     businesses: {
+      throttle: 200,
+      debounce: 200,
       query () {
         return getBusinessQuery(this.search_query, this.search_affiliations, this.search_tags)
       },
       variables () {
-        const query = this.search_query || undefined
+        let query = this.search_query || undefined
         const affiliations = this.search_affiliations.length > 0 ? this.search_affiliations.map(a => a.id) : undefined
         const tags = this.search_tags.length > 0 ? this.search_tags.map(t => t.id) : undefined
 
-        return this.minimum_search_criteria ? { query, affiliations, tags } : {}
+        if (query) query = query.trim().replace(/\s/g, '|')
```

Comment:
I think you can keep this as `const` and apply the trim and replace to the search query

Reply:
Ah right, it should remain an empty string. Done!